net/src/packet/display.rs

```diff
@@ -227,6 +227,27 @@ impl Display for BridgeDomain {
         write!(f, "{}", self.get_id())
     }
 }
+
+#[inline]
```

Comment:
not sure why you would inline here.  If anything we should mark `Display::fmt for PacketMeta` as `#[cold]`